packages/react-components/src/templates/__tests__/ManuscriptForm.test.tsx

```diff
@@ -617,6 +657,14 @@ it('resets form fields to default values when no longer visible', async () => {
   expect(preprintDoiTextbox).toHaveValue(preprintDoi);
   expect(publicationDoiTextbox).toHaveValue(publicationDoi);
 
+  const quickChecks = screen.getByRole('region', { name: /quick checks/i });
+
+  within(quickChecks)
+    .getAllByText('Yes')
+    .forEach((button) => {
+      userEvent.click(button);
+    });
+
```

Comment:
as mentioned in the comment above, we can delete this because we already set the quick checks to "Yes" in `defaultProps` that are being passed to `ManuscriptForm`